csaf/src/visitors/skip/mod.rs

```diff
@@ -48,15 +48,25 @@ impl<V: DiscoveredVisitor> DiscoveredVisitor for SkipExistingVisitor<V> {
         context: &Self::Context,
         advisory: DiscoveredAdvisory,
     ) -> Result<(), Self::Error> {
-        let name = match advisory
-            .distribution
-            .directory_url
-            .make_relative(&advisory.url)
-        {
-            Some(name) => name,
-            None => return Err(Error::Name),
+        let name = if let Some(_directory_url) = &advisory.distribution.directory_url {
+            match advisory
+                .distribution
+                .directory_url
+                .clone()
+                .unwrap()
```

Comment:
Please don't use a let binding, ignore the value, and then unwrap.

Also please take a look at similar cases like this one.